### PR TITLE
398 - row level security

### PR DESCRIPTION
**Pull request by VB-at-Bis**
Brief README to describe roles and permissions (discussion points)

### CODE REVIEW

File: db/deploy/tables/attachment.sql [hunk 1]
```diff
@@ -29,10 +29,23 @@ perform ccbc_private.grant_permissions('select', 'attachment', 'ccbc_auth_user')
...
+alter table ccbc_public.attachment force row level security;
+alter table ccbc_public.attachment enable row level security;
 
+do
+$policy$
+begin
+-- ccbc_auth_user RLS: can see and modify only its own records
+perform ccbc_private.upsert_policy('ccbc_auth_user_select_attachment', 'attachment', 'select', 'ccbc_auth_user', 'application_id in (select id from ccbc_public.applications where owner=(select sub from ccbc_public.session()))');
+perform ccbc_private.upsert_policy('ccbc_auth_user_insert_attachment', 'attachment', 'insert', 'ccbc_auth_user', 'application_id in (select id from ccbc_public.applications where owner=(select sub from ccbc_public.session()))');
+perform ccbc_private.upsert_policy('ccbc_auth_user_update_attachment', 'attachment', 'update', 'ccbc_auth_user', 'application_id in (select id from ccbc_public.applications where owner=(select sub from ccbc_public.session()))');
```
Comment (marcellmueller) [inline db/deploy/tables/attachment.sql:45]: Since owner is being deprecated should we use created_by?

**Comment by marcellmueller** [comment]
Nice job on this, it's working well. Just left comment about owner column, it's fine for now but being deprecated soon in another ticket.

File: db/deploy/tables/attachment.sql (hunk 1 above)
Comment (VB-at-Bis) [reply db/deploy/tables/attachment.sql:45]: `main` still has `owner` - https://github.com/bcgov/CONN-CCBC-portal/blob/main/db/deploy/tables/applications.sql

File: db/deploy/views/applications_view.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
+-- Deploy ccbc:views/applications_view to pg
+-- requires: tables/applications
+
+begin;
+
+CREATE OR REPLACE VIEW ccbc_public.applications_view AS SELECT * FROM ccbc_public.applications;
+
+comment on view ccbc_public.applications_view is 'View containing thelist of all respondents applications';
```
Comment (AntBush) [inline db/deploy/views/applications_view.sql:8]: Small typo here `View containing thelist` -> `View containing the list`

Comment (VB-at-Bis) [reply db/deploy/views/applications_view.sql]: Thanks for noticing! Fixed...

**Review by marcellmueller — APPROVED** [review]

**Pull request merged**